shared/database/repos/boxFlow.js

```diff
@@ -1,34 +1,37 @@
 const boxFlowSchema = require("../../models/boxflow");
+const {
+  PHONE_NUMBER
+} = process.env
 
 module.exports.saveBoxFlow = (boxFlow) => {
   return boxFlowSchema.create({
-    phoneNumber: "whatsapp:+573008794497",
+    phoneNumber: `whatsapp:${PHONE_NUMBER}`,
     ...boxFlow,
   });
 };
 
 module.exports.getByCategories = () => {
   return boxFlowSchema.aggregate([
-    { $match: { phoneNumber: "whatsapp:+573008794497" } },
+    { $match: { phoneNumber: `whatsapp:${PHONE_NUMBER}`, type: "EXPENSE" } },
```

Comment:
Hey, should I re-populate my data in order to support this 😭😢?

Reply:
Not Really, you will need to re-deploy your lambdas. Let me upload the last commit.